csrc/scheduler/utils.h

```diff
@@ -158,43 +158,50 @@ void parallelizeAllLike(
     int64_t pos = -1,
     std::vector<TensorView*> selected_tvs = {},
     const std::unordered_set<ParallelType>& selected_parallel_types = {},
-    bool propagate_padding = true);
+    bool propagate_padding = true,
+    bool parallelize_inputs = false);
```

Comment:
Can you code-comment this? Make sure you explain this is about fusion inputs or just tv's inputs.